web_drop_target/static/src/js/web_drop_target.js

```diff
@@ -27,44 +28,45 @@ odoo.define('web_drop_target', function(require) {
         },
 
         _on_drop: function(e) {
-            var drop_item = this._get_drop_item(e);
-            if(!drop_item) {
-                return;
-            }
-            jQuery(e.delegateTarget).removeClass(this._drag_over_class);
-            var reader = new FileReader();
-            reader.onloadend = this.proxy(
-                _.partial(this._handle_file_drop, drop_item.getAsFile())
-            );
-            reader.readAsArrayBuffer(drop_item.getAsFile());
+            var self = this;
+            var drop_items = this._get_drop_items(e);
+            _.each(drop_items, function(drop_item) {
```

Comment:
Cannot this loop be moved to _handle_file_drop (which could then be renamed _handle_drop_items or something similar) ?
The goal is to be able to process multiple file differently than creating an attachement, i.e. having one rpc call for all the items instead of one rpc call for each dropped item.
I'm actually trying to do something like that for v11 but am struggling with it.

Reply:
I could finally get to it, cf : https://github.com/OCA/web/pull/975/commits/87be514256265bca13a55e285233043213acae9b
Maybe there's a cleaner solution ?

Reply:
@hbrunn seems good to me, can you check it?

Reply:
the contributor made it clear [not to want to waste time on my review](https://github.com/OCA/web/pull/971#discussion_r199557546), so I'm done with this PR

Reply:
@grindtildeath I cherry-picked your commit and modified it to make it works at v10. But now the stored files are empty. 
Can you please, check if it's also happening at your PR? or it's just me that I've done something wrong

Reply:
No need to check anything. I didn't see this commit [db9e332](https://github.com/OCA/web/pull/975/commits/db9e332f3b2fcd7a566a9075d5ffda0de5566302). Applied and now works properly